engine/src/main/scala/cromwell/engine/language/CromwellLanguages.scala

```diff
@@ -0,0 +1,44 @@
+package cromwell.engine.language
+
+import cromwell.engine.language.CromwellLanguages._
+import cromwell.engine.workflow.lifecycle.materialization.LanguageFactory
+
+/**
+  * Provides a global singleton access to the instantiated language factories.
```

Comment:
`global singleton` made me think this was going to be an `object`. Could we move that part on the `Instance` below that actually represents the global singleton ?